descriptions/css/properties/animation-duration.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "animation-duration": {
+        "__short_description": "The <strong><code>animation-duration</code></strong> <a href='https://developer.mozilla.org/docs/CSS'>CSS</a> property sets the length of time that an animation takes to complete one cycle."
```

Comment:
So this is an interesting one. I stripped the localization part of the URL (`en-US`), but otherwise left the URL intact. I'd like to have a generalized approach to something like this. Should we fix the URL on the wiki page? Or should I be checking every internal link for redirects on ingestion?